### PR TITLE
Update dependencies to kube 1.12

### PR DESCRIPTION
**Pull request by ash2k**

### CODE REVIEW

File: pkg/speccheck/speccheck.go [hunk 1]
```diff
@@ -102,9 +92,73 @@ func (sc *SpecCheck) compareActualVsSpec(spec, actual *unstructured.Unstructured
...
+	}
+}
+
+// TODO Is this ok? Check that there is only one controller and it is THIS bundle
+func processOwnerReferences(spec, actual []meta_v1.OwnerReference) []meta_v1.OwnerReference {
+	if len(spec) == 0 {
+		// return nil slice to make it go away
+		return nil
+	}
+	return spec
```
Comment (nilebox) [inline pkg/speccheck/speccheck.go]: Why do we pass `actual` if we never use it?
Is this function properly implemented?

File: pkg/speccheck/speccheck.go [hunk 2]
```diff
@@ -102,9 +92,73 @@ func (sc *SpecCheck) compareActualVsSpec(spec, actual *unstructured.Unstructured
...
+}
+
+// TODO Nukes added labels. Should be configurable per-object and/or per-object kind?
+func processLabels(spec, actual map[string]string) map[string]string {
```
Comment (nilebox) [inline pkg/speccheck/speccheck.go]: `actual` not used

File: pkg/speccheck/speccheck.go (hunk 1 above)
Comment (nilebox) [reply pkg/speccheck/speccheck.go]: So we need to keep non-controller ownerReferences?

Comment (ash2k) [reply pkg/speccheck/speccheck.go]: I've removed unused arguments.

**Pull request merged**